### PR TITLE
Update readme to reflect multiple examples integrated

### PR DESCRIPTION
**Pull request by da-kami**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,13 +12,15 @@ Set up a local development environment for COMIT apps with one command.
 
 ## 2 - Create your first project!
 
-1. Create [hello-swap](https://github.com/comit-network/hello-swap/) app: `yarn create comit-app new hello-swap`,
-2. Start environment (blockchain nodes and COMIT nodes): `yarn create comit-app start-env`
-3. Run hello-swap (in separate terminal):
-    1. Navigate to the example app directory `cd <path-to-hello-swap>`
+1. `yarn create comit-app new <your-app-name>`,
+2. `yarn create comit-app start-env` (start blockchain nodes and COMIT nodes)
```
Comment (luckysori) [inline README.md]:
```suggestion
2. `yarn create comit-app start-env` to start blockchain nodes and COMIT nodes,
```

File: README.md [hunk 2]
```diff
@@ -12,13 +12,15 @@ Set up a local development environment for COMIT apps with one command.
...
+3. Run the [btc-eth](https://github.com/comit-network/create-comit-app/tree/master/new_project/examples/btc-eth) example (in separate terminal):
+    1. Navigate to the btc-eth example app directory `cd <path-to-your-app>/examples/btc-eth`
```
Comment (luckysori) [inline README.md]:
```suggestion
    1. Navigate to the btc-eth example app directory `cd <path-to-your-app>/examples/btc-eth`,
```

File: README.md [hunk 3]
```diff
@@ -12,13 +12,15 @@ Set up a local development environment for COMIT apps with one command.
...
     2. `yarn install` (or `npm install`) to install dependencies
```
Comment (luckysori) [inline README.md]:
```suggestion
    2. `yarn install` (or `npm install`) to install dependencies,
```

File: README.md [hunk 4]
```diff
@@ -12,13 +12,15 @@ Set up a local development environment for COMIT apps with one command.
...
     3. `yarn start` (or `npm start`) to run the application
```
Comment (luckysori) [inline README.md]:
```suggestion
    3. `yarn start` (or `npm start`) to run the application.
```

File: README.md [hunk 5]
```diff
@@ -12,13 +12,15 @@ Set up a local development environment for COMIT apps with one command.
...
 
+You find additional examples in the [examples](https://github.com/comit-network/create-comit-app/tree/master/new_project/examples) directory that is created as part of step 1.
```
Comment (luckysori) [inline README.md]:
```suggestion
You can find additional examples in the [examples](https://github.com/comit-network/create-comit-app/tree/master/new_project/examples) directory that is created as part of step 1.
```

File: README.md (hunk 3 above)
Comment (thomaseizinger) [inline README.md]: We only ship a `yarn.lock` file. That doesn't work with `npm`. We should either ship a `package-lock.json` file or limit our instructions to `yarn`.

File: README.md (hunk 1 above)
Comment (D4nte) [inline README.md]:
```suggestion
2. `yarn create comit-app start-env` to start blockchain and COMIT nodes,
```

File: README.md (hunk 2 above)
Comment (D4nte) [reply README.md]:
```suggestion
    a. Navigate to the btc-eth example app directory `cd <path-to-your-app>/examples/btc-eth`,
```

File: README.md (hunk 3 above)
Comment (da-kami) [reply README.md]: since the description including `npx` adds unnecessary complexity I decided to remove the whole `npm`/`npx` part.

**Pull request merged**